src/components/CardList.jsx

```diff
@@ -1,6 +1,6 @@
 import Card from "./Card";
 
-const CardList = ({ links }) => {
+const CardList = ({ links, folderName, folderList }) => {
   if (!links.length) {
     return (
```

Comment:
개인적으로는 조건은 명시적으로 작성해주는게 좋다고 생각하고 있어서 이렇게 작성해주시면 어떨까 싶네요~
```suggestion
  if (links.length === 0) {
    return (
```